src/components/SendAsset/SendAsset.js

```diff
@@ -175,19 +175,14 @@ const SendAsset = ({
       return;
     }
 
-    // $FlowFixMe
     const transactionPayload: TransactionPayload = {
       to: selectedContact.ethAddress,
-      // $FlowFixMe: flow update to 0.122
       receiverEnsName: selectedContact.ensName,
       amount: truncateAmount(currentValue.toString(), assetData.decimals),
-      // $FlowFixMe: bignumber.js types
       txFeeInWei: feeInfo.fee,
       // $FlowFixMe: flow update to 0.122
-      symbol: assetData.token,
-      // $FlowFixMe: flow update to 0.122
+      symbol: assetData.symbol || assetData.token,
```

Comment:
Side noter: I think we really really need to get rid of any remaining `token` references :( or even replace `AssetData` usage with just `Asset`

Reply:
I definitely agree, Asset vs AssetData vs AssetOption vs Collectible is a bit confusing atm. Let's plan that for the future. Also would suggest clarify the naming, e.g. `Asset = (Fungible)Token | NonFungibleToken` or similar.

Reply:
👍